satpy/composites/ahi.py

```diff
@@ -15,30 +13,8 @@
 #
 # You should have received a copy of the GNU General Public License along with
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
-"""Composite classes for the AHI instrument."""
+"""Composite classes for AHI adjustments."""
```

Comment:
Adjustments? This module is for AHI specific composites, even if it only has the green correction it is still a composites module. Not a huge deal, but it stood out to me.

Reply:
Fixed in d053f22e.